packages/valory/skills/oracle_abci/skill.yaml

```diff
@@ -122,6 +122,8 @@ models:
       service_id: price_estimation
       service_registry_address: null
       setup:
+        all_participants:
+        - {}
```

Comment:
Suggest to set a single zero address

Reply:
aee585279e78ca23e328cf4428015809f2ddfe69